src/store/actions.ts

```diff
@@ -6,23 +6,15 @@
  * @see https://vuex.vuejs.org/guide/structure.html
  */
 
+/*
 import { ActionContext } from 'vuex';
 import RootState from './RootState';
-import {
-	SET_LEMMA,
-	SET_LEXICAL_CATEGORY,
-} from './mutations';
 
 type RootContext = ActionContext<RootState, RootState>;
+ */
 
-export const UPDATE_LEMMA = 'updateLemma';
-export const UPDATE_LEXICAL_CATEGORY = 'updateLexicalCategory';
+// export const ACTION_NAME = 'actionName';
 
 export default {
-	[ UPDATE_LEMMA ]( { commit }: RootContext, lemma: string ): void {
-		commit( SET_LEMMA, lemma );
-	},
-	[ UPDATE_LEXICAL_CATEGORY ]( { commit }: RootContext, lexicalCategory: string ): void {
-		commit( SET_LEXICAL_CATEGORY, lexicalCategory );
-	},
+	/* [ ACTION_NAME ]( { commit }: RootContext, payload ) { ... } */
```

Comment:
Or should we leave this out and rely on the git history next time?

Reply:
I guess we should fully rely on git and bring back the example we had here before, with the delayed increment

Reply:
But that means also bringing back the counter in the root state, which doesn’t seem useful…

Reply:
Good point. I guess just more context is warrented, that is, to make it clear that actions are to be used for async operations. and in conjunction with imported mutations.

Reply:
Alright, I’ve added a comment in that direction and removed the commented-out code.

Reply:
Apologies, I didn't make myself clear enough. I meant to add the context alongside the commented usage examples, this way in the diff of adding the first action, it would be more visible for comparison.

Reply:
I think you were clear, I just decided for myself that the commented-out code was redundant with the Git history :P but your point about code review when adding the next action is valid, so I can add it back in.